context.go

```diff
@@ -169,6 +169,22 @@ func valueResult(ctx *Context, rtn C.RtnValue) (*Value, error) {
 	return &Value{rtn.value, ctx}, nil
 }
 
+func valueStrings(ctx *Context, rtn C.RtnStrings) ([]string, error) {
+	if rtn.strings == nil {
+		return []string{}, newJSError(rtn.error)
+	}
+	length := rtn.length
+	slice := (*[1 << 28]*C.char)(unsafe.Pointer(rtn.strings))[:length:length]
+	var result []string
```

Comment:
Since we know the size of `result`, we should be able to preallocate:

```go
	result := make([]string, len(slice))
	for i := 0; i < len(slice); i++ {
		s := slice[i]
		defer C.free(unsafe.Pointer(s))
		result[i] = C.GoString(s)
	}
	defer C.free(unsafe.Pointer(rtn.strings))
	return result, nil
```